packages/ozone/src/mod-service/index.ts

```diff
@@ -295,6 +299,11 @@ export class ModerationService {
         .where('modTool', 'is not', null)
         .where(sql`("modTool" ->> 'name')`, 'in', modTool)
     }
+    if (ageAssuranceState) {
+      builder = builder
+        .where('action', '=', 'tools.ozone.moderation.defs#ageAssuranceEvent')
+        .where(sql`meta->>'status'`, '=', ageAssuranceState)
```

Comment:
I notice `action` isn't usually specified in these queries.  Is it needed here because `meta->>'status'` is used for multiple action types?